test/ssr/compile-with-webpack.ts

```diff
@@ -3,6 +3,9 @@ import webpack from 'webpack'
 import MemoryFS from 'memory-fs'
 
 export function compileWithWebpack (file, extraConfig, cb) {
+  if( process.env.FOOBAR !== 'blah') {
+    throw new Error('die')
+  }
```

Comment:
remove this :D

Reply:
Hahah oh yeah, forgot I got desperate while picking apart the builds ;)